src/ValueValidators/Error.php

```diff
@@ -5,22 +5,30 @@
 namespace ValueValidators;
 
 /**
- * @license GPL-2.0+
+ * @license GPL-2.0-or-later
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
 class Error {
 
 	public const SEVERITY_ERROR = 9;
 	public const SEVERITY_WARNING = 4;
 
+	/** @var string */
 	private $text;
+
+	/** @var int */
 	private $severity;
+
+	/** @var ?string */
 	private $property;
 
+	/** @var string */
 	private $code;
+
+	/** @var array */
```

Comment:
Why not add PHP types instead of comments?

Reply:
totally fair - changed that now.